### ISSUE TITLE
[YSQL] yb-server-ctl.sh tserver stop fails to kill some PIDs

### ISSUE THREAD

**Issue by timothy-e** [yugabyte/yugabyte-db#19819]
Jira Link: [DB-8754](https://yugabyte.atlassian.net/browse/DB-8754)
### Description

Output from `yb-server-ctl.sh tserver stop`:
```
Waiting on PID: 14626
Found postmaster PIDs: 14697
Wait for 30s...
Found invalid Postgres PID: 594. Skipped.
Found invalid Postgres PID: 1982. Skipped.
Found invalid Postgres PID: 3346. Skipped.
Found invalid Postgres PID: 4783. Skipped.
Found invalid Postgres PID: 6388. Skipped.
Found invalid Postgres PID: 6464. Skipped.
Found invalid Postgres PID: 6790. Skipped.
Found invalid Postgres PID: 6812. Skipped.
Found invalid Postgres PID: 7849. Skipped.
Killing postgres PID: 10484...
Killing postgres PID: 11238...
Killing postgres PID: 14697...
Killing postgres PID: 14815...
Killing postgres PID: 15122...
Killing postgres PID: 15190...
Killing postgres PID: 15196...
Killing postgres PID: 15239...
Killing postgres PID: 31081...
Stopped.
```

Notice that all 3 and 4 digit PIDs were skipped, and 5 digit PIDs were killed. I believe this is because of whitespace (that isn't printed for some reason). I can reproduce the problem with a subset of the script locally, and fix it by adding 

```
pg_pid=`echo $pg_pid | sed 's/ *$//g'`
```

inside the loop on pg_pid. 

### Issue Type

kind/bug

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

[DB-8754]: https://yugabyte.atlassian.net/browse/DB-8754?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by timothy-e** [comment]
Duplicate of #19817
